moduleframework/common.py

```diff
@@ -444,26 +445,29 @@ def getModulemdYamlconfig(self, urllink=None):
                       can be overridden by the **CONFIG** envvar.
         :return: dict
         """
-        try:
-            if urllink:
-                ymlfile = urllib.urlopen(urllink)
-                cconfig = yaml.load(ymlfile)
-                link = cconfig
-            elif not get_if_module():
-                trans_dict["GUESTPACKAGER"] = "yum -y"
-                link = {"data": {}}
+        link = {"data": {}}
+        if urllink:
+            ymlfile = urllib.urlopen(urllink)
+            modulemd = yaml.load(ymlfile)
```

Comment:
This can be moved to alone short function. Later on it is called again.

Reply:
could be. Is there any reason why to excluede it to separated function?